config/locales/application_form.yml

```diff
@@ -28,3 +28,21 @@ en:
         label: Email address
       address:
         label: Postal address
+
+    degree_details_section:
+      heading:
+        Academic qualifications
+      type:
+        label: Type of degree
+        hint_label: For example, BA, BSc, Masters, Phd, or other (please specify).
+      subject:
+        label: Subject of degree
+      institution:
+        label: Institution where you studied
+      class:
+        label: Class of degree
+      year:
+        label: Year qualification was awarded
+      button:
+        add_degree: 'Add another degree'
```

Comment:
How come this entry is in quotes, but the others aren't? (my `.yml`-fu is not strong)